src/libs/formatPhoneNumber.js

```diff
@@ -0,0 +1,61 @@
+import lodashGet from 'lodash/get';
```

Comment:
We typically don't create utility functions in their own files. can you move this to `UserUtils.js`? https://github.com/Expensify/App/blob/main/src/libs/UserUtils.js

Reply:
Ok, so maybe I shouldn't have removed the LocalePhoneNumber util (I did it because we're not longer using its functions). Maybe instead of moving it to the user util it would be better to restore LocalePhoneNumber and move the function there?